posts/inside-rust/2024-11-04-test-infra-oct-2024-2.md

```diff
@@ -0,0 +1,168 @@
+---
+layout: post
+title: "This Month in Our Test Infra: October 2024"
+author: Jieyou Xu
+team: the Bootstrap Team <https://www.rust-lang.org/governance/teams/infra#team-bootstrap>
+---
+
+# This Month in Our Test Infra: October 2024
+
+<!-- time period: 2024-10-09 through 2024-11-04 -->
+
+This is a quick summary of the changes in the test infrastructure for the [rust-lang/rust][r-l/r]
+repository[^scope] for **October 2024**[^spec-date]. It also includes brief descriptions of on-going
+work.
+
+[^scope]: The test infra here refers to the test harness [compiletest] and supporting components in
+our build system [bootstrap]. This test infra is used mainly by rustc and rustdoc. Other tools like
+cargo, miri or rustfmt maintain their own test infra.
+[^spec-date]: specifically 2024-10-09 to 2024-11-04. Note that the previous issue incorrectly used
+    October in the filename and thus URL, but is actually the September issue.
+
+As usual, if you encounter bugs or UX issues when using our test infrastructure, please [file an
+issue][new-issue]. Bugs and papercuts can't be fixed if we don't know about them!
+
+**Thanks to everyone who contributed to our test infra!**
+
+## Highlights
+
+### `compiletest` now supports bringing your own custom diff tool
+
+[`compiletest` (and bootstrap) now supports bringing your own custom diff tool
+(#131181)][custom-diff-tool].
+
+![A ui test failure showing stderr snapshot diff produced by a custom differ](../../../images/2024-11-04-test-infra-oct-2024-2/compiletest-custom-differ.png)
```

Comment:
This doesn't show up locally no matter how hard I try, but the other blog posts use this kind of relative path so...